packages/api-fetch/src/index.js

```diff
@@ -36,6 +36,10 @@ function apiFetch( options ) {
 			headers[ 'Content-Type' ] = 'application/json';
 		}
 
+		if ( ! headers.Accept ) {
```

Comment:
Minor: Rather than adding a bunch of `if` statements, we could simplify this using [`Object.assign`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) in assigning these as defaults:

```js
const headers = {
	Accept: 'application/json, */*;q=0.1',
	'Content-Type': 'application/json',
	...remainingOptions.headers
};
```

Reply:
Makes sense, will update.